src/proxy-page/proxy-page.service.ts

```diff
@@ -55,6 +56,12 @@ export class ProxyPageService {
   ) {
     this.context.Init(spaceKey, pageId, theme, style);
     this.context.setTitle(data.title);
+    const version: Version = {
+      versionNumber: data.version.number,
+      lastModification: new Date(data.version.friendlyWhen),
```

Comment:
Is it mandatory to have lastModification and modificationBy for our needs?

Reply:
Finally, we don't need to retrieve the version for our basic need (certificate), but i agreed with @PiiXiieeS , we decided to return an object that contains (versionNumber, lastModification, modificationBy) and maybe in the futur we will add other feelds .